ml_service/util/create_scoring_image.py

```diff
@@ -29,7 +29,7 @@
 os.chdir("./code/scoring")
 
 image_config = ContainerImage.image_configuration(
-    execution_script="score.py",
+    execution_script=e.score_script,
```

Comment:
Need to add this VAR to the pipeline variables template now with default value of `score.py`

Reply:
It is in azdo-variables.yml